app.py

```diff
@@ -45,6 +46,9 @@
 from helpers.tooltips import tool_tips_kz, tool_tips_kz_nm
 from src.demo_enums import ProblemType
 
+log = logging.getLogger('werkzeug')
+log.setLevel(logging.ERROR)
```

Comment:
I think I would slightly prefer doing something like this just so we're not suppressing anything during development (DEBUG imported from config).

```suggestion
if not DEBUG:
    log = logging.getLogger('werkzeug')
    log.setLevel(logging.WARNING)
```